File: pmd/java/listas/listaex02/ex3/Circulo.java
package listas.listaex02.ex3;

public class Circulo implements Geometria {

  private double raio;

  public Circulo(double raio) {
    super();
    this.raio = raio;
  }

  @Override
  public double area() {
    return 3.1415 * raio * raio;
  }

  @Override
  public double comprimento() {
    return 2 * 3.1415 * raio;
  }

  public String toString() {
    return String.format(
        "\nCÍRCULO { raio: %.2f, área: %.2f, comprimento: %.2f }",
        raio, area(), comprimento());
  }

}

File: alg/05/circulo.c
#include<stdio.h>

struct ponto {
  float x, y;
};


struct circulo {
  struct ponto centro;
  float raio;
};

void main() {
  
  struct circulo c; 
  c.centro.x = 4.0; 
  c.centro.y = 3.0; 
  c.raio = 2.5;

  printf("Centro do circulo: (%.1f, %.1f)\n", 
    c.centro.x, c.centro.y);

  float area = 3.14 * c.raio * c.raio;

  printf("Area do circulo: %.1f\n", area);

}
